mail/dovecot2-sieve/Portfile

```diff
@@ -1,74 +1,51 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github                      1.0
+PortGroup           cxx11                       1.1
+
+github.setup        dovecot pigeonhole 0.5.6
 
 name                dovecot2-sieve
-version             0.5.0.1
-# set hg.tag to tag or rev.
-hg.tag              ${version}
-#hg.tag              2027
+version             0.5.6
```

Comment:
Is this needed ? version should be automatically set via the github.setup line ?

Reply:
Whoops forgot to comment.

Thanks. I deleted this line.